### PR TITLE
Add risk assessment

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: domain.yml [hunk 1]
```diff
@@ -283,6 +287,50 @@ responses:
   utter_submit:
     - text: "All done!"
 
+  utter_risk_low:
+    - text: |
+        <insert-low-risk-image-uuid> 💚
```
Comment (miltontony) [inline domain.yml:292]: Issue created to replace this with the actual image #10

**Comment by miltontony** [comment]
@rudigiesler ready for review

File: actions/actions.py [hunk 2]
```diff
@@ -475,7 +486,21 @@ def submit(
...
+            [
+                (slot, tracker.get_slot(slot))
+                for slot in self.SLOTS
+                if slot.startswith("symptoms_")
+            ]
+            + [
+                ("exposure", tracker.get_slot("exposure")),
+                ("age", tracker.get_slot("age")),
+            ],
+        )()
```
Comment (rudigiesler) [inline actions/actions.py:500]: I'm not sure why we're creating a `partial`, but then executing it immediately. Could we not just directly call `dict`?

**Comment by miltontony** [comment]
Also closes #10

**Pull request merged**